### PR TITLE
Added plotting function for sobol indices of higher order

### PR DESCRIPTION
**Pull request by Simardeep27**
Currently, the <code>plot_sobol_indices()</code> method provides a plot for only 1st order sobol indices, 
Added the functionality for choosing the order of the indices (1, 2, 3) . Also providing a default parameters if none provided

### CODE REVIEW

File: equadratures/plot.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 import matplotlib
 from equadratures.datasets import score
 import numpy as np
+import scipy as sp
```
Comment (ascillitoe) [inline equadratures/plot.py:8]: I don't think scipy is needed here?

**Review by ascillitoe — CHANGES_REQUESTED** [review]
Hi @Simardeep27, this is looking good! The scipy import just needs to be removed and then we're good to go!

**Comment by Simardeep27** [comment]
Sorry about that, have changed it now

**Comment by ascillitoe** [comment]
Thanks!

**Pull request merged**